src/index.ts

```diff
@@ -580,7 +583,17 @@ export function main(argString = ""): void {
     withStash(stashItems, () => {
       withVIPClan(() => {
         // 0. diet stuff.
-        if (!globalOptions.noDiet) runDiet();
+        if (globalOptions.noDiet || property.getBoolean("_garboYachtzeeChainCompleted")) {
```

Comment:
```suggestion
        if (globalOptions.noDiet || get("_garboYachtzeeChainCompleted", false)) {
```

Reply:
I was in fact using `get` prior to this, but it seemed to be incorrectly parsing _ prefs for some reason. Forcing it with `getBoolean` seemed to resolve the issue for me - I can't really figure out what the underlying cause of the parsing error was